libs/types/src/cdf_cast_vote_records.ts

```diff
@@ -1112,6 +1136,25 @@ export const CandidateSelectionSchema: z.ZodSchema<CandidateSelection> =
     IsWriteIn: z.optional(z.boolean()),
   });
 
+/**
+ * Represents metadata about a batch (like it's label) and is linked to by
+ * a BatchId in {@link CVR}.
+ */
+export interface Batch {
+  '@type': 'CVR.vxBatch';
+  '@id': string;
+  BatchLabel?: string;
```

Comment:
If we're exporting it it should always have a label, but we have logic to handle batches without labels in VxAdmin so I wondered if it was possible. Looking through it again, I think that's only the case for GEMS results, and I can make this required here. It makes sense because if there's no label here, then it's not even useful data.

Reply:
Sounds good!